src/FunctionApps/python/IntakePipeline/fhir.py

```diff
@@ -0,0 +1,9 @@
+from typing import Iterator
+
+
+def read_fhir_bundles() -> Iterator[dict]:
+    yield {}
```

Comment:
Graham, is the intention for this function to provide an iterator over the actual converted FHIR bundles? My yield fu is weak right now w.r.t. to memory implications (I _think_ yield doesn't load anything until the calling for loop asks for the object), but will this work with our memory constraints? Or do we need to make this an iterator over file names and then open individual files in another function?

Reply:
basically - the idea here is that downstream, people just for a `for x in y:` and don't have to worry about it. In the background, yep, it's about memory efficiency.